roles/postgresql/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+# tasks file for postgresql
```

Comment:
I would like to see a comment mentioning & linking to the PUL postgresql role this is based on